### PR TITLE
feat: display Message V4 API connection logs

### PR DESCRIPTION
**Pull request by jgiovaresco**
## Issue

https://gravitee.atlassian.net/browse/APIM-2472

## Description

A small description of what you did in that PR.

**API with no logs**
image

**API with logs**
image

## Additional context

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-bztjoewbxp.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs/api-runtime-logs.component.spec.ts [hunk 1]
```diff
@@ -15,50 +15,206 @@
...
     fixture.detectChanges();
   };
 
-  describe('GIVEN the current page is the Runtime Logs page', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+    httpTestingController.verify();
+  });
+
+  describe('GIVEN there is no logs', () => {
```
Comment (julian-robotham-gravitee) [inline gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs/api-runtime-logs.component.spec.ts]:
```suggestion
  describe('GIVEN there are no logs', () => {
```

File: gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs/api-runtime-logs.component.spec.ts [hunk 2]
```diff
@@ -15,50 +15,206 @@
...
+      expect(fakeUiRouter.go).toHaveBeenCalledWith('management.apis.ng.runtimeLogs-settings');
+    });
+
+    it('should not display the list', async () => {
+      expect(await componentHarness.getRows()).toHaveLength(0);
+      expect(await componentHarness.getPaginator()).toBeNull();
+    });
+  });
+
+  describe('GIVEN there is logs', () => {
```
Comment (julian-robotham-gravitee) [inline gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs/api-runtime-logs.component.spec.ts]:
```suggestion
  describe('GIVEN there are logs', () => {
```

File: gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs/components/api-runtime-logs-empty/api-runtime-logs-empty.component.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+
+.empty-card {
+  text-align: center;
+
+  .icon {
+    border-radius: 30px;
+    background-color: mat.get-color-from-palette(gio.$mat-primary-palette, 'lighter');
```
Comment (julian-robotham-gravitee) [inline gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs/components/api-runtime-logs-empty/api-runtime-logs-empty.component.scss:10]: Thanks that's better than hard coded values

File: gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs/components/api-runtime-logs-list-row/api-runtime-logs-list-row.component.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  <div class="log-row__left">
+    <div class="log-row__line-1">
+      <div>{{ log.timestamp }}</div>
+      <span class="gio-badge-neutral">Request ended</span>
+    </div>
+
+    <div class="log-row__line-2">
+      <div class="log-row__line-2__request">
+        <div class="log-row__line-2__data">
+          <div class="property">Entrypoint:</div>
```
Comment (Okhelifi) [inline gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs/components/api-runtime-logs-list-row/api-runtime-logs-list-row.component.html]: As for the moment we don't have the entrypoint I would suggest to reword it to the following.

```suggestion
          <div class="property">Http method:</div>
```

**Review by Okhelifi — APPROVED** [review]
